core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiService.scala

```diff
@@ -40,10 +40,11 @@ trait AdminApiService extends HttpService with PerRequestCreator with UserInfoDi
     } ~
     path("admin" / "project" / "registration") {
       post {
-        parameters("project", "bucket") { (project, bucket) => requestContext =>
+        entity(as[RawlsBillingProjectTransfer]) { xfer =>
+        requestContext =>
           perRequest(requestContext,
             UserService.props(userServiceConstructor, userInfo),
-            UserService.AdminRegisterBillingProject(RawlsBillingProjectName(project), userInfo.userEmail, bucket))
+            UserService.AdminRegisterBillingProject(RawlsBillingProjectName(xfer.project), RawlsUserEmail(xfer.newOwner), xfer.bucket))
```

Comment:
This is what inspired this change: this endpoint has to be called by an admin, but we might want to reassign the project to a non-admin. (i.e. dumbledore to hermione)

Reply:
good call